scripts/extensions/pool_update.precheck

```diff
@@ -140,7 +141,7 @@ def execute_precheck(session, control_package, yum_conf_file, update_precheck_fi
             for msg in livepatch_messages.keys():
                 if msg in precheck_output:
                     return livepatch_messages[msg]
-            raise PrecheckFailure()
+            raise PrecheckFailure('Unknown precheck result: %s' % precheck_output)
```

Comment:
Thanks @mseri for the comment.
In fact I just re-test and find `re.sub('\n +', ' ', output, flags=re.DOTALL)` is redundant and it doesn't help the format of the output of CLI. Without it, the output is even cleaner.
So I will remove them.

Also with discussion with @srowe, to return `Unknown precheck result: ...` is inappropriate, we should check if the precheck script runs successfully, if no raise error, if yes but no matching livepatch_message, return `ok`.

I will update the commit.